src/loader.js

```diff
@@ -373,7 +373,7 @@ function pitch(request) {
   const { NodeTemplatePlugin } = webpack.node;
   const { NodeTargetPlugin } = webpack.node;
 
-  new NodeTemplatePlugin(outputOptions).apply(childCompiler);
+  new NodeTemplatePlugin().apply(childCompiler);
```

Comment:
@alexander-akait  We have only `filename` and `publicPath` in outputOptions but the [NodeTemplatePlugin only uses `asyncChunkLoading`](https://github.com/webpack/webpack/blob/8e0e0eb47d47fe33e4c9ef9bc455d9584f7ea92c/lib/node/NodeTemplatePlugin.js#L32). I think don't need to pass it here, TS also warns about the same.

<img width="751" alt="Screenshot 2023-06-15 at 6 14 48 AM" src="https://github.com/webpack-contrib/mini-css-extract-plugin/assets/80162912/bc792519-50e6-41aa-9ec5-e101f19103f2">